spec/requests/jobseekers/job_applications_spec.rb

```diff
@@ -48,7 +48,7 @@
 
           it "redirects to `new_quick_apply_jobseekers_job_job_application_path`" do
```

Comment:
Think we need to change this as it is redirecting to `about_your_application_jobseekers_job_job_application_path` now. Could add a test to assert that it redirects to `new_quick_apply_jobseekers_job_job_application_path` if the jobseeker has the right to work in the uk